includes/class-img-converter.php

```
@@ -344,30 +360,63 @@ private function is_animated_webp( $file ) {
+				$home_url    = untrailingslashit( home_url() );
+				$content_url = untrailingslashit( content_url() );
+				$local_base  = wp_normalize_path( ABSPATH );
+
+				if ( strpos( $source_image, $home_url ) === 0 ) {
+					$relative_path = str_replace( $home_url, '', $source_image );
+				} elseif ( strpos( $source_image, $content_url ) === 0 ) {
+					$relative_path = str_replace( $content_url, '', $source_image );
+					$local_base    = wp_normalize_path( WP_CONTENT_DIR );
+				} else {
+					$relative_path = $source_image;
+				}
```

PHPMD (2.15.0)

[error] The method get_img_path uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)